Week-01/Day-02_Finding-Nemo/finding_nemo_botiboti/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(non_snake_case)]
```

Comment:
We should use language conventions instead of insisting on the exercise's example names.